sentry/src/main/java/io/sentry/InvalidSentryTraceHeaderException.java

```diff
@@ -0,0 +1,17 @@
+package io.sentry;
```

Comment:
we have a `io.sentry.exception` package, wondering if we should move all the exception types in there

Reply:
👍

Reply:
I'd be in to move InvalidDsnException too, so we do a single breaking change, as `performance` by itself is already one.

Reply:
👍